mail-client/geary/geary-43.0.ebuild

```diff
@@ -0,0 +1,100 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{8..11} )
```

Comment:
We can actually drop this and the `inherit python-any-r1` because of this:

````diff
-# Post-install scripts
-meson.add_install_script('build-aux' / 'post_install.py')
-
-# GNOME Builder doesn't support YAML manifests, so generate a JSON
-# version from the YAML and commit it. :( GNOME/gnome-builder#520
-yaml_to_json = find_program('build-aux' / 'yaml_to_json.py')
-custom_target(
-  'org.gnome.Geary.json',
-  build_by_default: true,
-  command: [yaml_to_json, '@INPUT@', meson.source_root(), '@OUTPUT@'],
-  input: files('org.gnome.Geary.yaml'),
-  output: 'org.gnome.Geary.json'
+# Post-install script
+gnome.post_install(
+  gtk_update_icon_cache: true,
+  glib_compile_schemas: true,
+  update_desktop_database: true,
 )
````